### PR TITLE
Crossplane blog

### PR DESCRIPTION
**Pull request by martyjhenderson**
This PR:
- Adds the Crossplane blogpost
- Adds the images needed
- Updates the readme to reflect the need for Ruby 2

### CODE REVIEW

File: _posts/2022-08-05-crossplane-demo.md [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+date:        2022-08-05 09:00:00
+author:      "Marty Henderson"
+description: ""
+headerImg:  "/assets/images/posts/2022-08-05-crossplane/popsicle-header.jpg"
+cross-post-link: "https://nalth.is/crossplane"
+cross-post-text: "Nalthis"
+---
+
+
+Header Image - Photo by <a href="https://unsplash.com/es/@taylorheeryphoto?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Taylor Heery</a> on <a href="https://unsplash.com/s/photos/popsicle?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a>
```
Comment (nerddtvg) [inline _posts/2022-08-05-crossplane-demo.md:15]: Can you remove the utm parameters? We don't need to pass through tracking data.

This isn't the header image despite the var name, it is actually the thumbnail on the blog post list. Not sure if you want to rephrase that.

File: _posts/2022-08-05-crossplane-demo.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+  
+_This post originally appeared on Marty Henderson's [personal blog](https://nalth.is/crossplane)._
+
+[Crossplane](crossplane.io) is a newer member of the Infrastructure-as-Code team, with many components still yet to be built or in an alpha or beta state. Although it is new, it introduces some novel ideas of how to manage infrastructure that you deploy, especially if you are familiar with Kubernetes
```
Comment (nerddtvg) [inline _posts/2022-08-05-crossplane-demo.md:19]: The Markdown URL requires a scheme, this is currently redirecting to `/2022/08/05/crossplane.io`

File: _posts/2022-08-05-crossplane-demo.md [hunk 3]
```diff
@@ -0,0 +1,231 @@
+---
+layout:      posts
+background:  shortBackground
+title:       "Crossplane, a Kubernetes-powered IaC Tool"
+subtitle:    ""
+date:        2022-08-05 09:00:00
```
Comment (nerddtvg) [inline _posts/2022-08-05-crossplane-demo.md:6]: Once this is approved, you'll want to update this date before merging.

File: _posts/2022-08-05-crossplane-demo.md [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+
+!["Using kubectl to describe the AKS Cluster"](/assets/images/posts/2022-08-05-crossplane/describe-akscluster.png)
+
+Alright, there's a lot of notes in here, but let's skip down to the relevant part of this object.
+
+!["AKS Status Successful"](/assets/images/posts/2022-08-05-crossplane/aks-status-state.png)
+
+So, we're cooking with gas now! You can even verify in the Azure console if you really want.
+
+!["Status in Azure Console of AKS"](/assets/images/posts/2022-08-05-crossplane/aks-in-azure-console.png)
```
Comment (nerddtvg) [inline _posts/2022-08-05-crossplane-demo.md:203]: Azure Portal* to stick with branding terms

File: _posts/2022-08-05-crossplane-demo.md [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+
+So, now that it's in state, what if I don't want it anymore?
+
+Delete the yaml file!
+
+As you can guess by the fact I have the DigitalOcean provider and configuration, I had some DigitalOcean resources. However, that Provider is still maturing and would created a brand new Droplet every time the status of configuration was checked - 8 droplets later, I hastily deleted the yaml file and applied it and the Droplets stopped being created. The DO Provider is still not good at deleting resources, but the AWS one is!
```
Comment (nerddtvg) [inline _posts/2022-08-05-crossplane-demo.md:209]: Any issue replacing "DO" with "DigitalOcean"? You use the full name twice in the paragraph and then the abbreviation. The abbreviation wasn't used elsewhere except a token variable.

**Review by nerddtvg — CHANGES_REQUESTED** [review]

File: _posts/2022-08-05-crossplane-demo.md (hunk 1 above)
Comment (martyjhenderson) [reply _posts/2022-08-05-crossplane-demo.md:15]: True, in my blog post it was the header, I'll just say "image" and drop the tracking.

File: _posts/2022-08-05-crossplane-demo.md (hunk 3 above)
Comment (martyjhenderson) [reply _posts/2022-08-05-crossplane-demo.md:6]: Updating this as I go along

File: _posts/2022-08-05-crossplane-demo.md (hunk 4 above)
Comment (martyjhenderson) [reply _posts/2022-08-05-crossplane-demo.md:203]: Fixed (though not in the image name)

File: _posts/2022-08-05-crossplane-demo.md (hunk 2 above)
Comment (martyjhenderson) [reply _posts/2022-08-05-crossplane-demo.md:19]: Fixed

File: _posts/2022-08-05-crossplane-demo.md (hunk 5 above)
Comment (martyjhenderson) [reply _posts/2022-08-05-crossplane-demo.md:209]: Good call, corrected

**Review by nerddtvg — APPROVED** [review]

**Pull request merged**